build/BranchInfo.props

```diff
@@ -1,8 +1,25 @@
 <Project>
   <PropertyGroup>
+    <StableProjects>
+        Microsoft.ML;
+        Microsoft.ML.CpuMath;
+        Microsoft.ML.FastTree;
+        Microsoft.ML.ImageAnalytics;
+        Microsoft.ML.LightGbm;
+        Microsoft.ML.Mkl.Components;
+        Microsoft.ML.Mkl.Redist;
+    </StableProjects>
+    <IsStableProject Condition="'$(MSBuildProjectName.Contains(.symbols))' == 'false'">$(StableProjects.Contains($(MSBuildProjectName)))</IsStableProject>
+    <IsStableProject Condition="'$(MSBuildProjectName.Contains(.symbols))' == 'true'">$(StableProjects.Contains($(MSBuildProjectName.Substring(0, $(MSBuildProjectName.IndexOf(.symbols))))))</IsStableProject>
     <MajorVersion>0</MajorVersion>
     <MinorVersion>12</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <PreReleaseLabel>preview</PreReleaseLabel>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(IsStableProject)' == 'true'">
```

Comment:
One other place that uses these versions is the native build. In that world we generate a `version.h` or `version.c` file during the build using these properties.

We may also need to check for the `Native` build here as well.

Reply:
You can see it here:

https://github.com/dotnet/buildtools/blob/4642e67685b9c21c4d5160ea7665843a48ed8155/src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets#L102-L108

Reply:
@eerhardt could you please elaborate more on the actionable item here?

Reply:
We probably won't be able to separate the native assets from non-"1.0" and "1.0" without a decent amount of native build work. Honestly, it wouldn't be worth it IMO, as the native version is more for debugging/informational purposes. It isn't as important as the NuGet package version or the AssemblyVersion.

So I'd say let's call the native assets "stable" in this work. When we ship "1.0", all the native assets will be versioned as "1.0". The only native assembly we build that may not be in a stable package is the `MatrixFactorizationNative` assembly. But again, there isn't much value in having that native assembly versioned as non-"1.0".